packages/safe-apps-provider/README.md

```diff
@@ -0,0 +1,67 @@
+# Safe Apps Provider
+
+[![npm](https://img.shields.io/npm/v/@gnosis.pm/safe-apps-provider)](https://www.npmjs.com/package/@gnosis.pm/safe-apps-provider)
+
+This is a provider that follows common standards (e.g. [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193)) and can be used with various Web3 libraries (e.g. Web3js or Ethers)
+
+### How to use
+
+- Add npm package
+
+```bash
+yarn add @gnosis.pm/safe-apps-provider
+
+npm i @gnosis.pm/safe-apps-provider
+```
+
+### SafeAppsProvider
+
+- Usage example with [safe-apps-react-sdk](https://github.com/gnosis/safe-apps-sdk/tree/master/packages/safe-apps-react-sdk) and [Ethers.js](https://docs.ethers.io/v5/api/providers/other/#Web3Provider)
```

Comment:
Took me some time to realize that the difference between those snippers are about web3/ethers, maybe add a heading to both of them like `Usage example with ethers.js`